Algorithms.Tests/Encoders/NYSIISEncoderTest.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Algorithms.Encoders;
+using NUnit.Framework;
+using NUnit.Framework.Internal;
+
+namespace Algorithms.Tests.Encoders
+{
+    public static class NYSIISTests
+    {
+        [Test]
+        public static void AttemptNYSIIS()
+        {
+            string[] names = new string[] {
```

Comment:
Codacy found an issue: [Remove the array type; it is redundant.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6311443)